share/spice/quote_of_the_day/quote_of_the_day.css

```diff
@@ -0,0 +1,57 @@
+.quote__of__the__day--container {
```

Comment:
All CSS rules should be prefixed with `.zci--quote_of_the_day` to ensure it only applies inside our IA's div.

I think these CSS classes can be simplified: `quote__container, quote__text, quote__attribution`